partition/roles/metal-core/files/bgp-neighbor-stats.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+tmpfile=$(mktemp /var/run/bgp-neighbors/bgp-neighbors.XXXXXX)
+destfile=/var/run/bgp-neighbors/bgp-neighbors.json
+
+/usr/bin/vtysh -c "show ip bgp vrf all neighbors json" > "${tmpfile}"
+
+rm -f "${destfile}"
```

Comment:
```suggestion
rm -f "${destfile}"

tmpfile=$(mktemp /var/run/bgp-neighbors/bgp-neighbors.XXXXXX)
destfile=/var/run/bgp-neighbors/bgp-neighbors.json

/usr/bin/vtysh -c "show ip bgp vrf all neighbors json" > "${tmpfile}"
```
`mktemp` or `vtysh` may fail.

Reply:
What exactly are you suggesting here, don't get it ?

Reply:
The same reason as above:
> If the script fails for any reason, metal-core still believes there are BGP neighbors. Please remove the destination file at the **beginning** of the script.

Reply:
But if anything fails, the mv will not happen, also metal-core checks for mtime <= 1 hour

Reply:
Thank you, I didn't know about that.